arch/risc-v/src/mpfs/Kconfig

```diff
@@ -773,6 +773,35 @@ config MPFS_CANFD_DATA_BITRATE1
 	default 4000000
 	depends on MPFS_CANFD1
 
+comment "MSS CAN 2.0 Options"
+
+config MPFS_HAVE_MSS_CAN
```

Comment:
This can be just
config MPFS_MSS_CAN

"MPFS_HAVE" makes no sense, since every MPFS device has this.